proxy/nginx.conf

```diff
@@ -0,0 +1,74 @@
+worker_processes 2;
+daemon off;
+
+error_log stderr;
+
+http {
+  charset utf-8;
+  log_format cloudfoundry 'NginxLog "$request" $status $body_bytes_sent';
+  access_log /dev/stdout cloudfoundry;
+  default_type application/octet-stream;
+  include mime.types;
+  sendfile on;
+
+  tcp_nopush on;
+  keepalive_timeout 30;
+  port_in_redirect off; # Ensure that redirects don't include the internal container PORT - 8080
+
+  server {
+    listen {{port}};
+    root public;
+    index index.html index.htm Default.htm;
+  }
+
+  ## Gunicorn specs
+  server {
+    # use 'listen 80 deferred;' for Linux
+    # use 'listen 80 accept_filter=httpready;' for FreeBSD
+    listen {{port}};
+    client_max_body_size 4G;
```

Comment:
The default 4G from gunicorn  is awfully big for catalog. Can we start with 250m?

Reply:
Done.